source/beamline/AMPseudoMotorControl.cpp

```diff
@@ -593,6 +598,28 @@ AMAction3* AMPseudoMotorControl::createCalibrateAction(double oldValue, double n
 	return 0;
 }
 
+bool AMPseudoMotorControl::childrenConnected() const
+{
+	bool result = false;
+
+	int childCount = childControls().count();
+
+	if (childCount > 0) {
+		bool connected = true;
+
+		for (int i = 0; i < childCount && connected; i++) {
+			AMControl *child = childControlAt(i);
+
+			if ( !(child && child->isConnected()) )
+				connected = false;
+		}
+
+		result = connected;
+	}
+
+	return result;
```

Comment:
Either is fine.  If we think about this semantically, what does the connectivity of a pseudomotor even mean with no children?